README.md

```diff
@@ -1,65 +1,4 @@
 <!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
-Creates an ECS service.
```

Comment:
This also looks like terraform-docs in generating useful documentation.

Reply:
Erm...not sure what you're saying. I believe terraform-docs is also broken for TF12. I'll see what I can do...

Reply:
terraform-docs changed and also broke for tf12.